pygments/lexers/mcfunction.py

```diff
@@ -0,0 +1,272 @@
+"""
+    pygments.lexers.mcfunction
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for MCFunction and related languages.
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from pygments.lexer import RegexLexer, default, include, using
+from pygments.token import (Comment, Error, Keyword, Literal, Name, Number,
+                            Operator, Punctuation, String, Text, Token,
+                            Whitespace)
+
+
+__all__ = ['SNBTLexer', 'MCFunctionLexer']
+
+
+class SNBTLexer(RegexLexer):
+    """Lexer for stringified NBT, a data format used in Minecraft
+
+
+    .. versionadded:: ?
+    """
+
+    name = "SNBT"
+    url = ""
```

Comment:
Could you put an URL here, for consumption by https://pygments.org/languages/? (My first search result is https://minecraft.fandom.com/wiki/NBT_format.)